css/home.scss

```diff
@@ -591,16 +591,178 @@ permalink: /css/homestyle.css
 }
 
 .products-section {
-  background: #2c3e50;
+  background-color: #2c3e50;
+  background-image: linear-gradient(315deg, #2c3e50 15%, #455d75 95%);
+  color: #ecf0f1;
+  min-height: 80vh;
   padding: 100px 100px;
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  justify-content: start;
   align-items: center;
-  @media screen and (max-width:767px) {
-    padding: 0 10px;
+  @media (max-width:767px) {
+    padding: 0 1.5em;
   }
-  @media screen and (max-width: 539px) {
-    padding: 0;
+
+  .title {
+    font-size: 3em;
+  }
+
+  .segment {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    max-width: 2000px;
+    width: 100%;
+    text-align: start;
+    margin: 3em 0em;
+
+    .customer-quote-section {
+      align-self: center;
+      max-width: 700px;
+      width: 100%;
+      font-family: 'Libre Baskerville', serif;
+      margin: 3em 0;
+      @media (max-width: 950px) {
+        padding: 0 1em;
+      }
+
+      .quote-text {
+        font-size: 1.2em;
+
+        @media (max-width: 539px) {
+          font-size: 1.6em;
+        }
+      }
+    }
+
+    .problem {
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      justify-content: start;
+      align-self: start;
+      max-width: 600px;
+      width: 100%;
+      padding: 1.8em 0 0 1.8em;
+      z-index: 1;
+
+      &:after {
+        content: "";
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        top: 0%;
+        left: 0%;
+        background-image: linear-gradient(330deg, transparent 70%, hsl(324, 36%, 25%) 100%);
+        transform: skew(8deg);
+        z-index: -1;
+      }
+    }
+
+    .solution {
+      display: flex;
+      flex-direction: column;
+      justify-content: end;
+      align-self: end;
+      position: relative;
+      max-width: 600px;
+      width: 100%;
+      padding: 0 1.8em 1.8em 0;
+      z-index: 1;
+
+      &:after {
+        content: "";
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        top: 0%;
+        left: 0%;
+        background-image: linear-gradient(150deg, transparent 70%, hsl(139, 36%, 25%) 100%);
+        transform: skew(8deg);
+        z-index: -1;
+      }
+
+      a {
+        color: inherit;
+        text-decoration: rgba(236, 240, 241, 0.7) wavy underline;
+        margin: 1em 0;
+        align-self: flex-end;
+
+        span {
+          display: inline-block;
+          margin-left: 1em;
+        }
+
+        &:hover span {
+          transform: translatex(5px);
+          transition: 100ms;
+        }
+      }
+    }
+
+    .problem, .solution {
+      line-height: 1.6em;
+    }
+
+    .icon {
+      position: absolute;
+      color: #253545;
+      font-size: 20em;
+      opacity: .3;
```

Comment:
Her ville jeg tilføje `user-select: none;` for at undgå at man kommer til at selecte ikonerne når man prøver at highlighte tekst